### PR TITLE
RMMIS-6888: Disable Chrome autocomplete

### PR DESCRIPTION
**Pull request by traviscooperscdhhs**

### CODE REVIEW

**Comment by kjschulz** [comment]
:+1:

File: src/Field.jsx [hunk 1]
```diff
@@ -93,6 +93,7 @@ class Field extends React.Component {
     props.help = help;
     props.bsStyle = classNames({error: this.props.hasError});
     props.label = <FieldLabel {...this.props}/>;
+    props.autoComplete = false;
```
Comment (mallenscdhhs) [inline src/Field.jsx]: Not sure we want to make this the default for all fields. Lets make this configurable. Set the default value to `true`, and configure the specific fields in question to `false`.

File: src/Field.jsx [hunk 2]
```diff
@@ -93,6 +93,7 @@ class Field extends React.Component {
     props.help = help;
     props.bsStyle = classNames({error: this.props.hasError});
     props.label = <FieldLabel {...this.props}/>;
+    props.autoComplete = this.props.autoComplete !== undefined ? this.props.autoComplete : true;
```
Comment (mallenscdhhs) [inline src/Field.jsx]: just make `autoComplete` a default property on the Field class. no need to specify it here

Comment (traviscooperscdhhs) [inline src/Field.jsx]: doh

Comment (traviscooperscdhhs) [inline src/Field.jsx]: will do

**Pull request merged**